libs/vertexai/langchain_google_vertexai/_utils.py

```diff
@@ -176,6 +176,7 @@ def get_generation_info(
             "finish_reason": (
                 candidate.finish_reason.name if candidate.finish_reason else None
             ),
+            "avg_logprobs": candidate.avg_logprobs,
```

Comment:
is it exposed for all models now? should we check whether the attribute exists maybe?

Reply:
It's just for Gemini, is an attribute of the class of the candidates so idk we can check if the attribute exists or maybe doesn't include it if it's None

Reply:
yep, like at the line above, plaese

Reply:
Added the checks